core/src/value/function/ffi.rs

```diff
@@ -30,7 +30,13 @@ impl<'js> JsFunction<'js> {
     }
 
     pub unsafe fn into_js_value(self, ctx: Ctx<'_>) -> qjs::JSValue {
-        let obj = qjs::JS_NewObjectClass(ctx.ctx, Self::class_id() as _);
+        let proto = ctx
+            .globals()
+            .get::<_, Object>("Function")
+            .unwrap()
+            .get::<_, Object>("prototype")
+            .unwrap();
```

Comment:
Seems It will be better cache `Function.prototype` within context instead lookup it each time.

Reply:
Yes, that was what I was thinking to, not caching the object could in fact result in panics. The `Function` field on the global object is also writable. So a different value could be assigned to the `Function` field which would cause this to panic.